internal/validator/validator_interface.go

```diff
@@ -0,0 +1,5 @@
+package validator
+
+type ValidatorClient interface {
+	CheckPrerequsits() error
```

Comment:
can you use a spellchecker? I am sure there's plenty of extensions in VSCode for that.